tests/test_common.py

```diff
@@ -204,8 +204,10 @@ def test_asarray_copy(library):
     b = asarray(a, copy=None)
     assert is_lib_func(b)
     a[0] = 0.0
-    if library == 'cupy':
+    if library in ('cupy', 'dask.array'):
         # A copy is required for libraries where the default device is not CPU
+        # dask made a breaking change in 2024.12: copy=None copies
```

Comment:
```suggestion
        # dask changed behaviour of copy=None in 2024.12 to copy
```
IMHO not a breaking change